api/server/router/build/build_routes.go

```diff
@@ -37,7 +37,7 @@ func (e invalidParam) InvalidParameter() {}
 
 func newImageBuildOptions(ctx context.Context, r *http.Request) (*types.ImageBuildOptions, error) {
 	options := &types.ImageBuildOptions{
-		Version:        types.BuilderV1, // Builder V1 is the default, but can be overridden
+		Version:        types.BuilderBuildKit,
```

Comment:
If we do this, we'll probably want to switch this by API version

Reply:
Definitely!